robottelo/ui/contentviews.py

```diff
@@ -30,6 +30,18 @@ def go_to_filter_page(self, cv_name, filter_name):
         strategy, value = locators['contentviews.select_filter_name']
         self.click((strategy, value % filter_name))
 
+    def set_calendar_date_value(self, name, value):
+        """Set the input value of a date field and press the button to hide
+         the calendar popup panel"""
+        date_elm = self.wait_until_element(
+            locators['contentviews.calendar_date_input'] % name)
+        date_elm.clear()
+        if value:
```

Comment:
That is why we don't need all these statements and can use simple:
`text_field_update` method from base class